cluster/manifests/deployment-service/controller-rbac.yaml

```diff
@@ -124,6 +124,48 @@ subjects:
     kind: User
     name: zalando-iam:zalando:service:k8sapi-local_deployment-service-executor
 ---
+{{- if .Cluster.ConfigItems.deployment_service_executor_cdp_permissions }}
```

Comment:
Let's be explicit and use `{{- if eq .Cluster.ConfigItems.deployment_service_executor_cdp_permissions "true }}`